### PR TITLE
Latest posts with a couple blog posts

### LINKED ISSUE #124: Add the latest blog post to the homepage
- [How to show the latest post as the homepage?](https://talk.jekyllrb.com/t/how-to-show-the-latest-post-as-the-homepage/2199/3)

### PR DESCRIPTION
**Pull request by oleksis**
Add the “Blacker than your formatter” lead
First paragraph from About Us
Read More link
Fix #124

### CODE REVIEW

File: _data/locales/es.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ es:
     - Conferencias
     - Comunidad
   index:
-    lead: Bienvenidos a la Comunidad de desarrolladores de Black Python.
+    lead: Más negro que tu formateador
```
Comment (kjaymiller) [inline _data/locales/es.yml:11]: Does this translate with familiarity in Spanish/other languages?

File: index.html [hunk 2]
```diff
@@ -6,3 +6,33 @@
...
 {% assign t = site.data.locales[page.lang][page.lang] %} {% if page.title %} {% assign header = page.title %} {% else %} {% assign header = site.title %} {% endif %}
 <h1>{{ t.index.lead }}</h1>
+<br />
+<h2 id="what-is-black-python-devs">What is Black Python Devs</h2>
+<hr />
+
+<p>Black Python Devs is an online-first community for python developers of all skill levels that identify as black.</p>
+<p>Our goal is to:</p>
+<ol>
+  <li>Show the vast community of black Python developers that exist but whose voices are often not amplified in the
```
Comment (kjaymiller) [inline index.html]:
```suggestion
  <li>Create a sense of community amongst Black Developers globally.</li>
```

File: index.html [hunk 3]
```diff
@@ -6,3 +6,33 @@
 
 {% assign t = site.data.locales[page.lang][page.lang] %} {% if page.title %} {% assign header = page.title %} {% else %} {% assign header = site.title %} {% endif %}
 <h1>{{ t.index.lead }}</h1>
+<br />
+<h2 id="what-is-black-python-devs">What is Black Python Devs</h2>
+<hr />
+
+<p>Black Python Devs is an online-first community for python developers of all skill levels that identify as black.</p>
```
Comment (kjaymiller) [inline index.html]:
```suggestion
<p>Black Python Devs is an online-first community for python developers of all skill levels that identify as Black.</p>
```

File: index.html [hunk 4]
```diff
@@ -6,3 +6,33 @@
...
+<p>Our goal is to:</p>
+<ol>
+  <li>Show the vast community of black Python developers that exist but whose voices are often not amplified in the
+    community.</li>
+  <li>Support black Python developers in their careers by highlighting their expertise and creating mentorship
+    opportunities.</li>
```
Comment (kjaymiller) [inline index.html]:
```suggestion
  <li>Support Black Python developers with career development resources.</li>
```

File: index.html [hunk 5]
```diff
@@ -6,3 +6,33 @@
...
+  <li>Learn from those who are creating and organizing events and other groups with communities of black developers.</li>
```
Comment (kjaymiller) [inline index.html]:
```suggestion
  <li>Share the experiences of other groups with communities of Black developers.</li>
```

**Review by kjaymiller — CHANGES_REQUESTED** [review]
At least in English, when referring to Black people as a group, it is capitalized. This is something we need to standardize on.

for Reference https://www.cjr.org/analysis/capital-b-black-styleguide.php

File: _data/locales/es.yml (hunk 1 above)
Comment (oleksis) [reply _data/locales/es.yml:11]: Is something depends of the context. Is like "negro" or "Negro" similar idea like "black" and "Black"

For Spanish/other we will  translate anyway ?

**Pull request merged**